extensions/amp-apester-media/0.1/amp-apester-media.js

```diff
@@ -320,64 +320,57 @@ class AmpApesterMedia extends AMP.BaseElement {
             this.element.appendChild(iframe);
             handleCompanionAds(media, this.element);
           })
-          .then(() => {
-            return this.loadPromise(iframe).then(() => {
-              return vsync.mutatePromise(() => {
-                if (this.iframe_) {
-                  this.iframe_.classList.add('i-amphtml-apester-iframe-ready');
-                  const campaignData = media['campaignData'];
-                  if (campaignData) {
-                    const bottomAdOptions = campaignData['bottomAdOptions'];
-                    if (bottomAdOptions && bottomAdOptions.enabled) {
-                      this.hasBottomAd_ = true;
-                      const ampdoc = this.getAmpDoc();
-                      Services.extensionsFor(
-                        this.win
-                      )./*OK*/ installExtensionForDoc(ampdoc, AD_TAG);
-                      this.iframe_.contentWindow./*OK*/ postMessage(
-                        BOTTOM_AD_MESSAGE,
-                        '*'
-                      );
-                    }
+          .then(() => this.loadPromise(iframe))
+          .then(() =>
+            vsync.mutatePromise(() => {
+              if (this.iframe_) {
+                this.iframe_.classList.add('i-amphtml-apester-iframe-ready');
+
+                const campaignData = media['campaignData'];
+                if (campaignData) {
+                  const bottomAdOptions = campaignData['bottomAdOptions'];
+                  if (bottomAdOptions?.enabled) {
+                    this.hasBottomAd_ = true;
+                    const ampdoc = this.getAmpDoc();
+                    Services.extensionsFor(
+                      this.win
+                    )./*OK*/ installExtensionForDoc(ampdoc, AD_TAG);
                     this.iframe_.contentWindow./*OK*/ postMessage(
-                      /** @type {JsonObject} */ ({
-                        type: 'campaigns',
-                        data: campaignData,
-                      }),
+                      BOTTOM_AD_MESSAGE,
                       '*'
                     );
                   }
+
+                  this.iframe_.contentWindow./*OK*/ postMessage(
+                    /** @type {JsonObject} */ ({
+                      type: 'campaigns',
+                      data: campaignData,
+                    }),
+                    '*'
+                  );
                 }
-                let height = 0;
-                if (media && media['data'] && media['data']['size']) {
-                  height = media['data']['size']['height'];
-                }
-                if (height != this.height_) {
-                  this.height_ = height;
-                  if (this.random_) {
-                    this.attemptChangeHeight(height);
-                  } else {
-                    this.forceChangeHeight(height);
-                  }
+              }
+
+              const height = media?.['data']?.['size'] ?? 0;
```

Comment:
nit: semantically the same and won't expand on compilation
```suggestion
              const height = media?.['data']?.['size'] || 0;
```